packages/application-components/src/utils/default-form-buttons.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useIntl, MessageDescriptor, IntlShape } from 'react-intl';
+import {
+  PrimaryButton,
+  SecondaryButton,
+  IconButton,
+  BinLinearIcon,
+} from '@commercetools-frontend/ui-kit';
+import buttonMessages from './button-messages';
+import filterDataAttributes from './filter-data-attributes';
+
+type Label = string | MessageDescriptor;
+type Props = {
+  label: Label;
+  onClick: (event: React.SyntheticEvent) => void;
+  isDisabled: boolean;
+  dataAttributes: { [key: string]: string };
+  children?: never;
+};
+
+const commonDefaultProps = {
+  isDisabled: false,
+  dataAttributes: {},
+};
+
+const confirmDefaultProps: Pick<
+  Props,
+  'label' | 'isDisabled' | 'dataAttributes'
+> = {
+  label: buttonMessages.confirm,
+  ...commonDefaultProps,
+};
+
+const getFormattedLabel = (label: Label, intl: IntlShape) =>
+  typeof label === 'string' ? label : intl.formatMessage(label);
+
+const ConfirmButton = (props: Props) => {
+  const intl = useIntl();
+  return (
+    <PrimaryButton
+      label={getFormattedLabel(props.label, intl)}
+      onClick={props.onClick}
+      isDisabled={props.isDisabled}
+      {...filterDataAttributes(props.dataAttributes)}
+    />
+  );
+};
+ConfirmButton.displayName = 'ConfirmButton';
+ConfirmButton.defaultProps = confirmDefaultProps;
+
+const cancelDefaultProps: Pick<
+  Props,
+  'label' | 'isDisabled' | 'dataAttributes'
+> = {
+  label: buttonMessages.cancel,
+  ...commonDefaultProps,
+};
+
+const CancelButton = (props: Props) => {
```

Comment:
`FormSecondaryButton` normally but given we have two variants I suppose `FormCancelButton`.

Reply:
I'll still call it `FormSecondaryButton`, since the other one is technically not a SecondaryButton but an IconButton with a trash icon, specifically for deletion, which I'll call `FormDeleteButton`.